airflow/providers/sftp/decorators/sensors/sftp.py

```diff
@@ -52,7 +52,6 @@ def __init__(
         task_id: str,
         **kwargs,
     ) -> None:
-        kwargs.pop("multiple_outputs")
```

Comment:
Is this change break this decorator in case of Airflow < 2.9?

Maybe we should make it under the condition?

Reply:
You're right, I will do that in a separate PR once this one is merged, for now I will just revert it